src/features/courses/course-repository.ts

```diff
@@ -414,6 +451,19 @@ class CourseRepository {
         return result;
     }
 
+    async getQuestionsFromTopicId(options: GetQuestionRepositoryOptions): Promise<CourseWWTopicQuestion[]> {
+        const result = await CourseWWTopicQuestion.findAll({
+            where: {
+                courseTopicContentId: options.id,
+                active: true
+            },
+        });
+        if (_.isNil(result)) {
+            throw new NotFoundError('Questions requested from a non-existent topic');
```

Comment:
Technically, this could mean that there are just no questions, but I don't think it merits changing.

Reply:
Would no questions result in nil or empty array?

Reply:
you're right - I think this check is not enough, length === 0 instead? (And a more accurate error message)